source/docs/appendices/migration-guides.md

```diff
@@ -7,6 +7,140 @@ This area will contain the Migration steps to follow for upgrading your store to
+
+```diff
+input,
++.input-height,
+select {
+  height: 3.4rem;
+}
+```
+
+### New PasswordStrengthHint
+
+We included `<PasswordStrengthHint>` to detail the expected complexity of the password.
+
+If you overrode the following files, you will ned to add the two new components introduced by the feature in them :
+
+`_components.scss`
+
+```diff
++@import "~theme/components/atoms/Form/Input/PasswordStrengthHint/PasswordStrengthHint";
++@import "~theme/components/atoms/ProgressStatus/ProgressStatus";
+```
+
+`src/web/theme/components/atoms/Form/Input/index.js`
+
+```diff
++import PasswordStrengthHint from "./PasswordStrengthHint";
+
+export {
+  ...
++ PasswordStrengthHint,
+};
+```
+
+You can deactivate of the `PasswordStrengthHint` feature by adding the variable `FRONT_COMMERCE_WEB_PASSWORD_HINT_DISABLE=true` to your environment 
+
+If you overrode some forms with password creation/change (`RegisterForm`, `ChangeUserPasswordForm`, `PasswordReset`, `PasswordResetForm`), you can add the component this way
+
+```diff
++import PasswordStrengthHint from "theme/components/atoms/Form/Input/PasswordStrengthHint/PasswordStrengthHint";
+
+<Password 
+  name="password"
+  ...
+/>
++ <PasswordStrengthHint
++  formValuePath="password" // must equal the password name value 
++/>
+```
+
+### `passwordValidation` deprecation 
+
+The file `src/web/theme/components/atoms/Form/Input/Password/passwordValidation.js` is now deprecated, if you overrode it, you will ned  to override the password validity configuration in `src/web/theme/components/atoms/Form/Input/Password/passwordConfig.js`.
+
+See ??? for more details on the password field validation configuration.
+
+If you overrode some of the impacted components (`RegisterForm`, `ChangeUserPasswordForm`, `PasswordReset`, `PasswordResetForm`), use the new `src/web/theme/components/atoms/Form/Input/Password/passwordFieldValidator.js` instead of the deprecated `passwordValidation` as follow
+
+```diff
+-import {
+-  isPasswordValid,
+-  errorMessage,
+-  MIN_PASSWORD_LENGTH,
+-  MIN_CHAR_CLASSES,
+-} from "theme/components/atoms/Form/Input/Password/passwordValidation";
++import {
++  passwordValidationRules,
++  passwordValidationErrors,
++} from "theme/components/atoms/Form/Input/Password/passwordFieldValidator";
+
+...
+
+<Password 
+  name="password"
+- validations={{
+-   magentoPasswordRule: (_, value) => isPasswordValid(value),
+- }}
++ validations={passwordValidationRules}
+- validationError={intl.formatMessage(errorMessage, {
+-   minLength: MIN_PASSWORD_LENGTH,
+-   minClasses: MIN_CHAR_CLASSES,
+- })}
++ validationError={passwordValidationErrors}
+/>
+```
+
+### New features in `2.11.0`
+
+These new features may be relevant for your existing application:
+- `<Password>` fields now display an eye icon to show the password clearly
+- New component : `<PasswordStrengthHint>` to show hints of password's strength criterias to the user
+- New component : `<ProgressStatus>` to show a progressbar with a label
```

Comment:
```suggestion
- `<Password>` fields now display an eye icon to reveal the password
- New component: `<PasswordStrengthHint>` to show hints of password's strength criterias to the user
- New component: `<ProgressStatus>` to show a progressbar with a label
```

Depending on how the password page ended up being reworked, it could be good to add links to the relevant sections there from here too.